kubetest/build.go

```diff
@@ -71,6 +71,9 @@ func (b *buildStrategy) Build() error {
 		target = "bazel-release"
 	case "dind":
 		return dind.NewBuilder(util.K8s("kubernetes"), util.K8s("test-infra", "dind"), control).Build()
+	case "e2e":
+		//TODO(Q-Lee): we should have a better way of build just the e2e tests
+		target = "bazel-release"
```

Comment:
why bazel-release? don't check this in yet I guess?

Reply:
bazel-release builds tarballs kinda like quick-release. it's something like `bazel build //build:release-tars`

Reply:
There's no make target for just e2e tests, is there? We should consider adding one, because the conformance tests really don't care about building kubelet.

Reply:
I don't think there is. We should.